### PR TITLE
[Task-2] Setup Stylesheet

### PR DESCRIPTION
**Pull request by gutakk**
Resolve https://github.com/gutakk/react-surveyor/issues/2

## What happened 👀

- Install bootstrap
- Add bootstrap to application css
- Add button with bootstrap style
 
## Insight 📝

N/A
 
## Proof Of Work 📹

The button is styled by bootstrap.
Screen Shot 2564-06-03 at 18 30 09

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "@typescript-eslint/eslint-plugin": "4.22.1",
     "@typescript-eslint/parser": "4.22.1",
     "axios": "0.21.1",
+    "bootstrap": "^5.0.1",
```
Comment (carryall) [inline package.json:18]: @gutakk Can we use the specific version here?
```suggestion
    "bootstrap": "5.0.1",
```

File: src/App.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ function App(): JSX.Element {
   return (
     <div className="app">
       <header className="app-header">
+        <button className="btn btn-primary">Hello from Bootstrap</button>
```
Comment (carryall) [inline src/App.tsx:11]: @gutakk Do we add this just to test that the Bootstrap is working? And are we going to remove it later?

Comment (gutakk) [reply src/App.tsx:11]: Yes, It will be removed when I start working on sign in page.

File: package.json (hunk 1 above)
Comment (gutakk) [reply package.json]: Good catch. Fixed in 29c4f20f32035e12d3288aa64cebf485d012d326

**Review by rafayet-monon — APPROVED** [review]

**Review by carryall — APPROVED** [review]

**Pull request merged**